examples/app/spectrogram/waterfall.py

```diff
@@ -1,3 +1,5 @@
+''' A module file for creating a waterfall sample for the spectogram.
+'''
```

Comment:
same comment as `audio.py` I don't think a docstring on this internal module is needed